frontend/pages/queries/ManageQueriesPage/components/QueriesTable/QueriesTable.tsx

```diff
@@ -38,7 +33,7 @@ export interface IQueriesTableProps {
   isAnyTeamObserverPlus: boolean;
   router?: InjectedRouter;
   queryParams?: {
-    platform?: SelectedPlatform;
+    platform?: string; // which targeted platform to filter queries by
```

Comment:
Is there a reason we are making this more broad?

Reply:
It's been a couple weeks since I wrote this, and looking back it seems like the reason is mainly that calling this `string` reflects the actual possibilities here (user can put anything in the URL param that they like). Then, a little further down I'm [checking whether it's a valid queryable platform value](https://github.com/fleetdm/fleet/blob/237d3c47ed511455a736214ce81644133e0ea475/frontend/pages/queries/ManageQueriesPage/components/QueriesTable/QueriesTable.tsx#L111-L113), and setting to "all" if it's not, so the result is the same type, `SelectedPlatform`. I just [make type explicit](https://github.com/fleetdm/fleet/pull/24446/commits/749a2319a4909ddb9663e13df64a950c987e2d3a).